pkg/config/config/config.go

```diff
@@ -292,27 +292,8 @@ func NewRedisConfig(u string) (conf RedisConfig, err error) {
 
 // GetRedisConfig returns a
 func GetRedisConfig(v *viper.Viper, mainOpt *redis.UniversalOptions, key, ptr string) (conf RedisConfig, err error) {
-	var localOpt *redis.Options
-
-	localKey := fmt.Sprintf("%s.%s", key, ptr)
```

Comment:
We must keep the `localOpt`. It is used for the flags, like in `cozy-stack serve --lock-url=redis://localhost:6379/1`.